scripts/restart_tee.sh

```
@@ -0,0 +1,3 @@
+nitro-cli terminate-enclave --all
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)